masonite/request.py

```diff
@@ -29,6 +30,7 @@ class Request(Extendable):
         Extendable {masonite.helpers.Extendable.Extendable} -- Makes this class have the ability to extend another class at runtime.
     """
 
+    status = response_statuses
```

Comment:
do `response_statuses()` (as a function) which should return the dictionary of status code mappings